tests/test_models.py

```diff
@@ -84,3 +87,28 @@ def test_create_promotion(self):
         self.assertEqual(data.active, promotion.active)
 
     # Todo: Add your test cases here...
+    def test_create_a_promotion(self):
+        """It should Create a promotion and assert that it exists"""
+        promotion = Promotion(title="New Customer", promo_type=PromotionType.PERCENTAGE_DISCOUNT, active = True)
+        self.assertEqual(str(promotion), "<Promotion New Customer id=[None]>")
+        self.assertTrue(promotion is not None)
+        self.assertEqual(promotion.id, None)
+        self.assertEqual(promotion.title, "New Customer")
+        self.assertEqual(promotion.promo_type, PromotionType.PERCENTAGE_DISCOUNT)
+        self.assertEqual(promotion.active, True)
+        promotion = Promotion(title="New Customer", promo_type=PromotionType.AMOUNT_DISCOUNT, active=False)
+        self.assertEqual(promotion.active, False)
+        self.assertEqual(promotion.promo_type, PromotionType.AMOUNT_DISCOUNT)
+    
+    def test_add_a_promotion(self):
```

Comment:
Is it being used anywhere? Also how is it different from previous function test_create_a_promotion?

Reply:
In here, **test_add_a_promotion** uses promotion.create(), which means the promotion will be added into database, but **test_create_a_promotion** doesn't, test_create_a_promotion is used to check if we can create promotion properly.